libs/react/src/lib/form/input/input.tsx

```diff
@@ -9,25 +9,28 @@ import {
   TextInputProps,
 } from '../types'
 import { IndicatorType, validateClassName } from '@sebgroup/extract'
+import React from 'react'
 
```

Comment:
maybe we can remove this component as we are using formItem to wrap the input. so when user use this form element, they will use with the other components with green/react. i.e:

```
<Form>
  <FormItem>
    <Email />
  </FormItem>
   ...
</Form>
```

Reply:
Remove which component ? Input component ? I only see this 
![Capture](https://user-images.githubusercontent.com/92840841/156526096-b5a2c441-b3b8-43b0-8aa1-76b5ccda2ad9.PNG)